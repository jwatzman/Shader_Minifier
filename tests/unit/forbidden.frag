struct f {
	float f;
};

float someFun() {
	return 42.0;
}
