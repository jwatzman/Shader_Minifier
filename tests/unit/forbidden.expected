struct f{float f;};float t()
{
  return 42.;
}